test/expected/create_hypertable.out

```diff
@@ -288,6 +288,57 @@ NOTICE:  table "test_1dim" is already a hypertable, skipping
 select create_hypertable('test_schema.test_1dim', 'time');
 ERROR:  table "test_1dim" is already a hypertable
 \set ON_ERROR_STOP 1
+--test data migration
+create table test_schema.test_migrate(time timestamp, temp float);
+insert into test_schema.test_migrate VALUES ('2004-10-19 10:23:54+02', 1.0), ('2004-12-19 10:23:54+02', 2.0);
+select * from only test_schema.test_migrate;
+           time           | temp 
+--------------------------+------
+ Tue Oct 19 10:23:54 2004 |    1
+ Sun Dec 19 10:23:54 2004 |    2
+(2 rows)
+
+\set ON_ERROR_STOP 0
+--should fail without migrate_data => true
+select create_hypertable('test_schema.test_migrate', 'time');
+ERROR:  table "test_migrate" is not empty
+\set ON_ERROR_STOP 1
+select create_hypertable('test_schema.test_migrate', 'time', migrate_data => true);
+NOTICE:  adding NOT NULL constraint to column "time"
+NOTICE:  migrating data to chunks...
+ create_hypertable 
+-------------------
+ 
+(1 row)
+
+--main table should now be empty
+select * from only test_schema.test_migrate;
```

Comment:
can we also check `select * from test_schema.test_migrate;` to see that the data is still there?